staging/src/k8s.io/apiextensions-apiserver/pkg/registry/customresource/etcd.go

```diff
@@ -372,3 +342,55 @@ func scaleFromCustomResource(cr *unstructured.Unstructured, specReplicasPath, st
 
 	return scale, foundSpecReplicas, nil
 }
+
+type scaleUpdatedObjectInfo struct {
+	reqObjInfo         rest.UpdatedObjectInfo
+	specReplicasPath   string
+	statusReplicasPath string
+	labelSelectorPath  string
+}
+
+func (i *scaleUpdatedObjectInfo) Preconditions() *metav1.Preconditions {
+	return nil
```

Comment:
should return `i.reqObjInfo.Preconditions()`, right?

Reply:
yeah fixed.